src/main.rs

```diff
@@ -17,6 +18,11 @@ mod slack;
 #[database("db")]
 pub struct DbConn(PgConnection);
 
+#[get("/openapi")]
+pub async fn openapi() -> NamedFile {
+    NamedFile::open("openapi/openapi.yaml").await.unwrap()
+}
```

Comment:
Can you make this `/static/openapi.yaml`

Reply:
What about `/api/spec/openapi.yaml`? I don't want to use `static` as to not conflict with the frontend's `static/` dir